.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+## *Tips*
+- *Thank you very much for contributing to Apache Hudi.*
+- *Please review https://hudi.apache.org/contributing.html before opening a pull request.*
+
+## What is the purpose of the pull request
+
+*(For example: This pull request adds quick-start document.)*
+
+## Brief change log
+
+*(for example:)*
+  - *Modify AnnotationLocation checkstyle rule in checkstyle.xml*
+
+## Verify this pull request
+
+*(Please pick either of the following options)*
+
+This pull request is a trivial rework / code cleanup without any test coverage.
+
+*(or)*
+
+This pull request is already covered by existing tests, such as *(please describe tests)*.
```

Comment:
How about adding below content?

(or)

This change added tests and can be verified as follows:

*(example:)*

    - *Added integration tests for end-to-end*
    - *Added HoodieClientWriteTest to verify the change*
    - *Manually verified the change by running a job locally*

```

Reply:
> How about adding below content?
> 
> (or)
> 
> This change added tests and can be verified as follows:
> 
> _(example:)_
> 
> * _Added integration tests for end-to-end_
> * _Added HoodieClientWriteTest to verify the change_
> * _Manually verified the change by running a job locally_

👍 , done.